src/Microsoft.OpenApi.OData.Reader/Edm/ODataPathProvider.cs

```diff
@@ -993,5 +999,54 @@ private void AppendBoundOperationOnNavigationPropertyPath(IEdmOperation edmOpera
                 }
             }
         }
+
+        private void AppendBoundOperationOnOperationPath(IEdmOperation edmOperation, bool isCollection, IEdmEntityType bindingEntityType)
+        {
+            bool isEscapedFunction = _model.IsUrlEscapeFunction(edmOperation);
+
+            // only composable functions
+            var paths = _allOperationPaths.Where(x => x.LastSegment is ODataOperationSegment operationSegment
+                                                    && operationSegment.Operation is IEdmFunction edmFunction
+                                                    && edmFunction.IsComposable).ToList();
+
+            foreach (var path in paths)
+            {
+                if (path.LastSegment is ODataOperationSegment operationSegment)
```

Comment:
negate this statement and continue when true to reduce the arrow head style of the code here

Reply:
fixed